nat/src/stateful/mod.rs

```diff
@@ -644,15 +644,15 @@ impl StatefulNat {
     fn process_packet<Buf: PacketBufferMut>(&mut self, packet: &mut Packet<Buf>) {
         // TODO: What if no VNI
         let Some(src_vpc_id) = Self::get_src_vpc_id(packet) else {
-            warn!(
+            debug!(
```

Comment:
Ideally, the same here. If packets have no vpc discriminants, we should probably not even try to nat them ?
Also, shouldn't `Self::get_src_vpc_id(packet) `be replaced by just `packet.get_meta().dst_vpcd`? and 
`Self::get_src_vpc_id(packet)`  by  `packet.get_meta().src_vpcd`?
What is the benefit of those wrappers?